fedmsg_genacls.py

```diff
@@ -9,11 +9,125 @@
 import pprint
 import subprocess
 import os
+
+import arrow
+import fedora.client.fas2
 import fedmsg.consumers
 import moksha.hub.reactor
 
 
+gitolite_template = """
+@admins = {admins}
+@provenpackager = {provenpackager}
+@fedora-arm = {fedora_arm}
+@fedora-s390 = {fedora_s390}
+@fedora-ppc = {fedora_ppc}
+
+repo @all
+    -   VREF/update-block-push-origin = @all
+    RWC = @admins @fedora-arm @fedora-s390 @fedora-ppc
+    R = @all
+"""
+
+
+class GitolitePrefixConsumer(fedmsg.consumers.FedmsgConsumer):
+    """ This is the new-world consumer that handles things for
+    pagure-over-dist-git.
+    """
+    topic = '*'
+    interesting_topics = [
```

Comment:
I believe ``topic`` can be a list now, so we should be able to drop the ``interesting_topics`` in favor of ``topic`` directly